cli/data.go

```diff
@@ -235,7 +409,7 @@ func downloadBinary(ctx context.Context, client datapb.DataServiceClient, dst st
 }
 
 // TabularData downloads binary data matching filter to dst.
-func (c *AppClient) TabularData(dst string, filter *datapb.Filter) error {
+func (c *appClient) TabularData(dst string, filter *datapb.Filter) error {
```

Comment:
Any reason to keep this exported if we are making most things unexported?

Reply:
Nope! Good catch.